shared/data/constants.js

```diff
@@ -1,16 +1,33 @@
 const parseUserAgentString = require('../js/shared-utils/parse-user-agent-string')
 const browserInfo = parseUserAgentString()
 
+const trackerBlockingEndpointBase = 'https://staticcdn.duckduckgo.com/trackerblocking'
+
+function isMV3 () {
+    return chrome?.runtime.getManifest().manifest_version === 3
+}
+
 function getConfigFileName () {
     let browserName = browserInfo?.browser?.toLowerCase() || ''
 
     // clamp to known browsers
     if (!['chrome', 'firefox', 'brave', 'edg'].includes(browserName)) {
         browserName = ''
     } else {
-        browserName = '-' + browserName + (chrome?.runtime.getManifest().manifest_version === 3 ? 'mv3' : '')
+        browserName = '-' + browserName + (isMV3() ? 'mv3' : '')
     }
-    return `https://staticcdn.duckduckgo.com/trackerblocking/config/v2/extension${browserName}-config.json`
+    return `${trackerBlockingEndpointBase}/config/v2/extension${browserName}-config.json`
```

Comment:
not strictly related to this PR, but not all of the combinations exist, namely https://staticcdn.duckduckgo.com/trackerblocking/config/v2/extension-firefoxmv3-config.json which can happen at some point . I think we should fallback to extension-config in this case?

Reply:
Yeah I haven't changed the behaviour at all here (out of scope as that's config stuff). Not sure if that's been tested, but cc @kzar in case you hit that problem in MV3 with the remote configs.